sc/src/main/webapp/WEB-INF/jsp/tiles/views/hca-landing-page/data.jsp

```diff
@@ -9,21 +9,21 @@
     <hr>
 
     <p> Single Cell Expression Atlas provides access to the analysis results of
-        <fmt:formatNumber pattern="#,##0" value="${hcaExperimentsCount}"/> data sets from the Human Cell Atlas
-        and <fmt:formatNumber pattern="#,##0" value="${humanExperimentsCount}"/> data sets in <i>Homo sapiens</i>
+        <fmt:formatNumber pattern="#,##0" value="${hcaExperimentsCount}"/> data
```

Comment:
You can remove the `pattern` attribute if you just want to insert commas and you don’t want to do any rounding.

Reply:
This was code from v1 and I didn't pay much attention to it. Will simplify.

Reply:
In reality, it will be while til we hit quadruple digits in terms of experiment counts... 😄